test/libsolutil/JSON.cpp

```diff
@@ -26,7 +26,7 @@
 
 #include <boost/test/unit_test.hpp>
 
-using namespace std;
+
```

Comment:
Yeah, as mentioned before - just delete the `using namespace std;` instance, without replacing it with a newline.

Reply:
no new lines :P